department-of-veterans-affairs/caseflow
# Kev ma/appeals 36674

Resolves [APPEALS 36674](https://jira.devops.va.gov/browse/APPEALS-36674)

# Description
When a remove package task is active and the user is a Mail Team Supervisor, the page is readOnly. Created a serializer for correspondence tasks and updated HTTP call to include information.

## Acceptance Criteria

## Testing Plan
1. Login in Mail Team Supervisor user
2. Go to queue/correspondence and click on a correspondence case
3. In the Request package action dropdown, select "Remove package from caseflow"
4. Fill in the text box and hit confirm
5. Return to the same correspondence case page
6. Note that all the edit and save buttons are disabled, and the dropdown is not visible

# Frontend
## User Facing Changes

 BEFORE|AFTER
<img width="711" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/104021955/a6173769-ebd3-4d7e-a09d-88ae3f82ae67">|<img width="686" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/104021955/1b976d2b-5c14-481b-a663-a5ae319ff8aa">

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
